src/vm/gdbjit.cpp

```diff
@@ -1950,6 +1962,13 @@ bool NotifyGdb::BuildLineProg(MemBuf& buf, PCODE startAddr, TADDR codeSize, Symb
             prevFile = lines[i].fileIndex;
         }
 
+        // GCC don't use the is_prologue_end flag to mark the first instruction after the prologue.
```

Comment:
If LLDB sees two lines with the same native address, it thinks that the code was compiled by GCC and sets is_prologue_end flag. We want to avoid such random prologue end in the middle of a function.